File: frontend/src/app/data/FantasyTeam.ts
export class FantasyTeam{
  fantasyPlayerName: string;
  fantasyTeamGroup: string;
  solo: string;
  soloScore: number;
  jungle: string;
  jungleScore: number;
  mid: string;
  midScore: number;
  support: string;
  supportScore: number;
  hunter: string;
  hunterScore: number;
  totalTeamScore: number;


  constructor(fantasyPlayerName: string = "", fantasyTeamGroup: string = "", solo: string = "", soloScore: number = 0.0,
              jungle: string = "", jungleScore: number = 0.0, mid: string = "", midScore: number = 0.0,
              support: string = "", supportScore: number = 0.0, hunter: string = "", hunterScore: number = 0.0,
              totalTeamScore: number = 0.0) {
    this.fantasyPlayerName = fantasyPlayerName
    this.fantasyTeamGroup = fantasyTeamGroup
    this.solo = solo
    this.soloScore = soloScore
    this.jungle = jungle
    this.jungleScore = jungleScore
    this.mid = mid
    this.midScore = midScore
    this.support = support
    this.supportScore = supportScore
    this.hunter = hunter
    this.hunterScore = hunterScore
    this.totalTeamScore = totalTeamScore
  }
}

File: backend/src/main/kotlin/com/nicolaspetras/splfantasy/model/output/FantasyTeam.kt
package com.nicolaspetras.splfantasy.model.output

/**
 * An SPL Fantasy Team that was drafted by Fantasy Player that is part of one of the
 * Fantasy Team Groups
 */
data class FantasyTeam(
    val fantasyPlayerName: String,
    // TODO: Update to FantasyTeamGroupName from String
    val fantasyTeamGroup: String,
    val solo: String,
    val soloScore: Double,
    val jungle: String,
    val jungleScore: Double,
    val mid: String,
    val midScore: Double,
    val support: String,
    val supportScore: Double,
    val hunter: String,
    val hunterScore: Double,
    val totalTeamScore: Double
)
